awx/main/utils/ansible.py

```diff
@@ -97,3 +98,14 @@ def could_be_inventory(project_path, dir_path, filename):
     except IOError:
         return None
     return inventory_rel_path
+
+
+def read_ansible_config(project_path, variables_of_interest):
+    fnames = [ os.path.join(project_path, 'ansible.cfg'), '/etc/ansible/ansible.cfg']
+    parser = ConfigParser()
+    conf = parser.read(fnames)
+    values = {}
+    for var in variables_of_interest:
+        if var in conf['default']:
```

Comment:
will need to look into if it's possible to throw a KeyError or something like that

Reply:
from the config? We wouldn't want to throw the keyerror ourselves.

Reply:
```
result_traceback:
Traceback (most recent call last):
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/awx/main/tasks.py", line 1209, in run
    private_data_files=private_data_files)
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/awx/main/tasks.py", line 1481, in build_env
    config_values = read_ansible_config(job.project.get_project_path(), list(map(lambda x: x[1], path_vars)))
  File "/var/lib/awx/venv/awx/lib64/python3.6/site-packages/awx/main/utils/ansible.py", line 109, in read_ansible_config
    if var in conf['default']:
TypeError: list indices must be integers or slices, not str
```

don't know why

will work on next week

Reply:
s/default/defaults/, fixed.